DataType/PercentDataType.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Netdudes\DataSourceryBundle\DataType;
+
+use Netdudes\DataSourceryBundle\Query\FilterCondition;
+
+class PercentDataType extends AbstractDataType
+{
+    /**
+     * Return the identifier of the default sort method.
```

Comment:
Looks like this is coming from the interface. It will be probably gone while the refactoring. All of these types has these line. Should I hide it with inheritdoc or is it ok for you to leave it for now?